CODEOWNERS

```diff
@@ -324,7 +324,9 @@
 /include/mgmt/                            @nrfconnect/ncs-pluto
 /include/modem/                           @nrfconnect/ncs-modem
 /include/mpsl/                            @nrfconnect/ncs-dragoon
+/include/net/                             @nrfconnect/ncs-co-networking
 /include/net/azure_*                      @nrfconnect/ncs-cia @coderbyheart
+/include/net/download_client*             @nrfconnect/ncs-modem @nrfconnect/ncs-cia
```

Comment:
```suggestion
/include/net/download_client*             @nrfconnect/ncs-modem
```

Reply:
I don't think it makes sense that CIA has ownership of this library.

Reply:
ok